docs/dev-guides/2-creating-an-xrpk.md

```diff
@@ -28,10 +30,14 @@ XRPackages can be built from multiple asset types, including WebXR sites written
 
 For the purposes of this guide, a barebones WebXR scene built with ThreeJS is available at the link below. This scene contains tracked controllers and an animated blob.
 
-[`Click here to download the project files`](/my-first-xrpk.zip)
+[`Click here to download the project files`](/threejs-xrpk-demo.zip)
```

Comment:
Maybe this should be a separate repository, with the download link pointing to the repository zip?

Reply:
It doesn't need to be addressed in this PR, just a note.

Reply:
Sure! Is there already a repo which would fit this?

Reply:
I created these two:

https://github.com/avaer/threejs-demo
https://github.com/avaer/aframe-demo

Reply:
thanks -- I don't think I can fork them yet though because they're empty!

Reply:
Oh ok I'll add a readme!

Reply:
Added readmes, hopefully should be forkable now.

Reply:
Awesome, thanks -- PRs made and I've updated this branch to use link from github :)